cogs5e/utils/targetutils.py

```diff
@@ -77,7 +77,11 @@ async def definitely_combat(ctx: "AvraeContext", combat: "Combat", args: ParsedA
             contextargs = argparse(contextargs)
 
         try:
-            target = await combat.select_combatant(ctx, t, f"Select target #{i + 1}.", select_group=allow_groups)
+            target = await combat.select_combatant(
+                t,
+                f"Pick your {i+1}{['th','st','nd','rd','th'][(i // 10 % 10 != 1) * min(i % 10, 4)]} target.",
```

Comment:
`ordinal(i+1)`

Reply:
I made a PR to add an ordinal() function

Reply:
https://github.com/avrae/avrae/compare/master...TheReverendB:avrae:Add-ordinal()